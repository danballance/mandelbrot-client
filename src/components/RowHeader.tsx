import * as React from "react";
import { Grid, Header, Icon } from 'semantic-ui-react'

export const RowHeader: React.FC = () => {
  return (
    <Grid.Row textAlign="right">
      <Grid.Column only='computer' computer={8} ></Grid.Column>
      <Grid.Column computer={8} mobile={16} > 
        <Header as="h1" color="yellow">
          <Icon name="chart line" />
          <Header.Content>Mandlebrot API Client</Header.Content>
        </Header>
      </Grid.Column>          
    </Grid.Row>
  );
}
