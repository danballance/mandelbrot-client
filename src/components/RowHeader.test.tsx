import React from 'react';
import { shallow } from 'enzyme'
import { RowHeader } from './RowHeader'

describe('RowHeader', (): void => {
    it('renders correctly', (): void => {
        const rowHeader = shallow(<RowHeader />);
        expect(rowHeader).toMatchSnapshot();
    })
});
