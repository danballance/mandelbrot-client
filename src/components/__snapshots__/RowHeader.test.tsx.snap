// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RowHeader renders correctly 1`] = `ShallowWrapper {}`;
